json_templates.rdl_json_exposure1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "properties": {
+    "identifier": "https://datacatalog.worldbank.org/search/dataset/0064252/Central-Asia-exposure-dataset---Transport",
+    "title": "Central Asia Exposure Dataset - Transport",
+    "description": "Regional layer of transportation infrastructure in Central Asia (road, rail, bridges).",
+    "risk_data_type": "exposure",
+    "publisher": {
+        "name": "World Bank",
+        "email": "",
+        "url": "www.worldbank.org"
+        },
+    "version": 1,
+    "purpose": "Regional climate and disaster risk analysis for risk financing and risk management",
+    "project": "Strengthening Financial Resilience and Accelerating Risk Reduction in Central Asia - Regional risk assessment.",
+    "details": ""
```

Comment:
Fields that contain only whitespace should be omitted from the data. Otherwise, it's harder for users and tools to determine whether a data point is present in a dataset because they need to check both the existence of the field and whether it contains meaningful data.